Gemfile

```diff
@@ -28,3 +28,4 @@ gem 'valid_attribute', group: [:development, :test]
 gem 'shoulda-matchers', group: [:development, :test], require: false
 gem 'devise'
 gem 'foundation-rails'
+gem 'pry', group: [:development, :test]
```

Comment:
with Rails, you can utilize pry-rails - it has some additional niceties / conveniences for Rails console